index.js

```diff
@@ -16,35 +16,190 @@ const forbiddenModules = [
+    // Arrow functions are forced to put the return values on a new line, rather than inline
+    // with the argument and fat arrow itself, which is awkward syntax
+    // This rule, when fixing code will then cause problems with the max-length rule
+    // https://github.com/airbnb/javascript/issues/1584#issuecomment-335676788
+    'function-paren-newline': 'off',
+
+    // Prettier handles this
+    // https://eslint.org/docs/rules/implicit-arrow-linebreak
+    'implicit-arrow-linebreak': 'off',
+
+    // Prettier handles indent
+    // https://eslint.org/docs/rules/indent
+    indent: 'off',
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-indent.md
+    'react/jsx-indent': 'off',
+
+    // Functions that take many positional arguments can be difficult to work
+    // with and produce less maintainable APIs. When more than three arguments
+    // are needed, named arguments should be used.
+    'max-params': ['error', 3],
+
+    // Knowing how to use bitwise operators is rare and people who don't know how never will
+    // For the few instances where they are useful, this rule is a nuisance, so it's unnecessary
+    // https://eslint.org/docs/rules/no-bitwise
+    'no-bitwise': 'off',
+
+    // Simplifies code and reduces bugs
+    // https://eslint.org/docs/rules/no-lonely-if
+    'no-lonely-if': ['error'],
+
+    // Restrict certain modules from being used
+    'no-restricted-modules': ['error', {paths: forbiddenModules}],
+    'no-restricted-imports': ['error', {paths: forbiddenModules}],
+
+    // Reduces careless typo mistakes
+    // https://eslint.org/docs/rules/no-return-assign
+    'no-return-assign': ['error'],
+
+    // Prettier handles this
+    // https://eslint.org/docs/rules/object-curly-spacing
+    'object-curly-spacing': 'off',
+
+    // Spreading is better than complex Object.assign
+    // https://eslint.org/docs/rules/prefer-object-spread
+    'prefer-object-spread': ['error'],
+
+    // Alphabetical sorting is the most reliable way to make code consistently readable
+    // This goes for imports, props, objects, etc.
+
+    // This one is warn so it isn't disruptive during development
+    // https://eslint.org/docs/rules/sort-imports
+    'sort-imports': [
+      'warn',
+      {
+        ignoreCase: true,
+        memberSyntaxSortOrder: ['multiple', 'single', 'all', 'none'],
+      },
+    ],
+
+    // This enforces props are alphabetical on <Components />
+    // Reserved props should be sorted first (key, ref, etc.)
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-sort-props.md
+    'react/jsx-sort-props': [
+      'error',
+      {
+        ignoreCase: true,
+        reservedFirst: true,
+      },
+    ],
+
+    // This ensures deconstructed props/state, etc. are alphabetical
+    // https://github.com/mthadley/eslint-plugin-sort-destructure-keys
+    'sort-destructure-keys/sort-destructure-keys': 'error',
+
+    // Use the sort-imports rule
```

Comment:
Can you clarify this comment to something like: "// Use the sort-imports rule instead"?

Reply:
https://github.com/americanexpress/eslint-config-amex/pull/20